artifacts/definitions/Linux/Sys/LogHunter.yaml

```diff
@@ -0,0 +1,43 @@
+name: Linux.Sys.LogHunter
+author: "Matt Green - @mgreen27"
+description: |
+  This artifact enables grep of Linux, MacOS and Windows logs. 
+  Parameters include SearchRegex and WhitelistRegex as regex terms.  
+  I have also included a Path exclusion regex to improve performance and skip 
+  notorious locations like /proc.  
+  
+  NOTE: nosymlink feature of glob is set so unexpected results may occur if 
+  targetting includes symlink files.
+  
+parameters:
+  - name: TargetFiles
+    default: '/var/log/**'
+  - name: SearchRegex
+    description: "Regex of strings to search in log line."
+    default: ' POST '
+    type: regex
+  - name: WhitelistRegex
+    description: "Regex of strings to leave out of output."
+    default:
+    type: regex
+  - name: ExcludePathRegex
+    description: "Regex of paths to exclude."
+    default: '^/(proc|sys|run|snap)|\.journal$'
+    type: regex
+    
+sources:
+  - query: |
+      LET files = SELECT OSPath FROM glob(globs=TargetFiles,nosymlink=True)
+        WHERE NOT if(condition=ExcludePathRegex,
```

Comment:
This does not exclude the paths. We need to add a recursion callback

Reply:
OK no worries: Ill add recursion_callback=RecursionCallback